packages/jmix-react-core/src/app/Router.tsx

```diff
@@ -34,11 +34,13 @@ function exec(re: RegExp, str: string, keys: pathToRegexp.Key[]) {
  * @param replace - use `history.replaceState(...)` instead of `history.pushState(...)`
  * @param title - browser tab title
  */
-export function redirect(to: string, replace = false, title = '') {
-  const currentFullPath = window.location.href.substr(window.location.origin.length);
+export function redirect(to: string, replace = false, title = "") {
+  const currentFullPath = window.location.href.substr(
+    window.location.origin.length
+  );
   if (currentFullPath === to) return;
 
-  if (currentRoute.hashMode) to = '#' + to;
+  if (currentRoute.hashMode) to = "#" + to;
```

Comment:
I suggest either to enforce single quote, or to allow both single and double quote.
@web-devel WDYT?

Reply:
I vote for allowing both